packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -157,12 +188,23 @@
   &:hover {
     .vertical-chevron-container {
       calcite-icon {
-        color: var(--calcite-color-text-1);
+        color: var(--calcite-input-date-picker-icon-color-hover, var(--calcite-color-text-1));
       }
     }
   }
 }
 
+.input {
+  --calcite-input-background-color: var(--calcite-input-date-picker-input-background-color);
+  --calcite-input-border-color: var(--calcite-input-date-picker-input-border-color);
+  --calcite-input-corner-radius: var(--calcite-input-date-picker-input-corner-radius);
```

Comment:
Does this also need to be applied to the `date-picker` or does it need its own CSS prop? cc @macandcheese

Reply:
Would that be a separate effort for the `date-picker` component itself? If I'm understanding...